### PR TITLE
Continuous Integration on Windows

### PR DESCRIPTION
**Pull request by trieloff**
There are more than 16 test failures right now, many seem path-related or otherwise Windows-specific, so I'll need a windows VM to fix them.

### CODE REVIEW

File: test/testGitUtils.js [hunk 1]
```diff
@@ -82,7 +82,12 @@ describe('Testing GitUtils', () => {
     assert.equal(await GitUtils.isDirty(testRoot, GIT_USER_HOME), true);
   });
 
-  it('isDirty #unit with submodules', async () => {
+  it('isDirty #unit with submodules', async function test() {
```
Comment (trieloff) [inline test/testGitUtils.js]: https://github.com/adobe/helix-testutils can help with that.

Comment (tripodsan) [reply test/testGitUtils.js]: my IDE doesn't llike the `condit`... ;-)

Comment (tripodsan) [reply test/testGitUtils.js]: but the problem is, when the test is skipping itself, the `afterEach` is not executed anymore; which can lead to side effects...

Comment (tripodsan) [reply test/testGitUtils.js]: see https://github.com/mochajs/mocha/issues/3740

**Pull request merged**